.eslintrc

```diff
@@ -35,7 +35,7 @@ rules:
   computed-property-spacing: [2, "never"]
 
   space-unary-ops: [2, {"words": true, "nonwords": false}]
-  space-after-keywords: [2, "always"]
+  keyword-spacing: [2, {"before": true, "after": true}]
```

Comment:
@MoOx I need your confirmation that this is the correct configuration. When I upgraded the package the configuration changed to this style. Please confirm that this is correct.